docs/deploy.md

```diff
@@ -548,6 +548,167 @@ This exposes a JSON-RPC interface to the Hardhat Network. Use the address, **htt
+
+4. Click **transact**. The Terminal panel displays a success status for the interaction with your smart contract.
+
+![sucessmessage](/img/docs/deploy/successmessage.png)
+
+### Interaction with Sample Contract 
+
+To illustrate interaction with smart contracts, the **EonWorkspace** is created in Remix. This workspace provides a set of script files to perform specific tasks, such as displaying information or executing a transaction within the blockchain.
+
+In the **Remix File Explorer**, the scripts are located in the **examples** folder. To execute a script, select the desired filename.js file. Right-click to open the drop-down menu. Select **Run**. 
+
+![eonworkspace](/img/docs/deploy/remix-eonworkspace.png)
+
+
+### Get Basic Information
+
+The script, get basic information.js displays the basic information that includes:
+
+* Block Height
+* Block 
+* Chain ID
+* Gas Price 
+
+The **Remix Terminal** displays the results. 
+
+![basicinfo](/img/docs/deploy/basic-info-remix.png)
+
+### Deploy a Contract
+
+The script, **deploy a contract.js** displays the contract address. 
+
+The MetaMask wallet connected with Remix appears with a notification of the contract deployment. Click **Confirm** to proceed with the deployment.
+
+![deployconfirm](/img/docs/deploy/metamask-deploy-confirm.png)
+
+The **Remix Terminal** displays the results. 
+
+![deploycontract](/img/docs/deploy/deploy-contract-remix.png) 
+
+### Interact with Existing Contracts 
+The script, **Interact with existing contracts.js** displays the wallet address invoking a contract.
+
+The MetaMask wallet connected with Remix appears with a notification of the payable transaction of ZEN (TZEN). Click **Confirm** to proceed with the transaction.
+
+![interactconfirm](/img/docs/deploy/metamask-interact-confirm.png)
+
+The **Remix Terminal** displays the results. 
+
+![remixinteract](/img/docs/deploy/interact-contract-remix.png)
+
+The MetaMask wallet connected with Remix appears with a second notification of the payable transaction of **ZEN (TZEN)**. Click **Confirm** to complete the transaction.
+
+![interactcomplete](/img/docs/deploy/metamask-interact-complete.png)
+
+### EOA to EOA transaction 
+
+The script, **Make EOA to EOA transactions.js** displays an EOA to EOA transaction.
+
+The MetaMask wallet connected with Remix appears with a notification of the pending transaction of sending **ZEN (TZEN)** to the receiving address. Click **Confirm** to proceed with the transaction.
+
+![eoaconfirm](/img/docs/deploy/metamask-eoa-confirm.png)
+
+The **Remix Terminal** displays the results. 
+
+![eoaeoa](/img/docs/deploy/eoa-eoa-remix-ok.png)
+
+The MetaMask wallet appears with a second notification of the **Send** transaction of **ZEN (TZEN)** to the receiving address. 
+
+![eoasend](/img/docs/deploy/metamask-eoa-send.png)
+
+### Interact with Native Smart Contracts
+
+The script, **Interact with native smart contracts.js** displays a backwards transfer (native contract). 
```

Comment:
Is `Interact with a native smart contracts.js` also a file that is provided by default?

Reply:
It would be nice to link to the Backward Transfer terminology. Right now there is no way to link to specific terms within the Terminology section.

```suggestion
The script, **Interact with native smart contracts.js** displays a [backwards transfer](/terminology) contract (native contract). 
```